### PR TITLE
Update JPA Class Transform Bytecode Collector Trace Spec to 'JavaPers…

### PR DESCRIPTION
**Pull request by jgrassel**
The trace spec enabling the logging of class bytecode before and after enhancement, 'JPA.BytecodeCapture', undesirably triggers with the "JPA=all" trace spec.  Changing the trace  specification to 'JavaPersistenceClassTransformation' with this change, in order to enable it only when specifically intended.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

File: dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAConstants.java [hunk 1]
```diff
@@ -17,10 +17,10 @@
 /**
  * Manifested Constants used by the JPA Service implementation.
  */
-public final class JPAConstants
-{
+public final class JPAConstants {
     // ********** JPA general, trace and bundling constants
     public static final String JPA_TRACE_GROUP = "JPA";
+    public static final String JPATRANSFORMER_TRACE_GROUP = "JavaPersistenceClassTransformation";
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAConstants.java]: Can we change this to 
```java
JPAConstants.JPABYTECODE_TRACE_GROUP = "JavaPersistenceByteCode";
```

Comment (jgrassel) [reply dev/com.ibm.ws.jpa.container.core/src/com/ibm/ws/jpa/management/JPAConstants.java]: Certainly

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
#run-libby-bot

**Pull request merged**